dspace/config/modules/versioning.cfg

```diff
@@ -14,3 +14,7 @@ versioning.item.history.view.admin=false
 # the submitter of a version should be included in the version history of
 # an item.
 versioning.item.history.include.submitter=false
+
+# If you want to allow submitters to create new versions of there items, set
+# the property submitterCanCreateNewVersion true.
+versioning.submitterCanCreateNewVersion=true
```

Comment:
Looking good overall. I wonder however if this option should be commented out by default?  At the very least, we also need to make it clear here that this is a *JSPUI ONLY* feature at this time. Because it's a feature specific to one UI, I also do wonder if we should comment it out by default... starting with DSpace 7 (new UI), perhaps we enable this by default.